src/en/juju-backups.md

```diff
@@ -0,0 +1,405 @@
+backup will be able to connect to and control your environments, so a further 
+step of encrypting this backup file may be advisable.
+ 
+### Restoring ~/.juju
+
+Restoring your Juju client settings and environments is a simple matter of 
+extracting the archive you created above.
+
+```
+tar xzf juju-yymmdd-hhmmss.tar.gz 
+```
+
+In the case of any local environment, this will also be restored providing the 
+actual service itself (LXC) has remained running.
+
+## The Juju state server (bootstrap node)
+
+Juju provides commands for recovering the Juju state server (bootstrap
+node) in case of failure. Included commands allow you to create, restore and 
+manage backup file containing the state server
+configuration, keys, and environment data. If the state server or
+its host machine later fails, you can create a new state server from the
+backup file. For environments with "High availability" enabled, also see the 
+[relevant sections below](#ha-(high-availability))
+
+### Creating a backup file
+
+Use the `create` command to create a new backup file:
+
+```bash
+juju backups create [-e ENV] [--quiet ] [--filename=FILENAME | --no-download]
+```
+
+The `create` command creates an archive file for the state server in an
+environment, along with metadata about that file.  By default the current
+environment is used. It may be explicitly identified by the -e option
+or the `JUJU_ENV` shell environment variable.
+
+As a convenience, the state server stores both the file and the
+metadata (use `juju backups list` to see stored files), however, it is up to the
+user to download and safely store these files for when they are needed. 
+By default `juju backups create`will download the new backup file, though you
+can use the --no-download option to disable this.
+
+If the --filename option is not provided to `create` then the backup
+archive is downloaded with a default name that incorporates the date and
+time, such as `juju-backup-20151103-1408.tar.gz`. The filename is displayed
+once the backup completes.  Unless the --quiet option is used, the backup
+metadata is also output.
+
+Examples:
+
+```bash
+juju backups create 
+juju backups create --no-download
+juju backups create -e my-env
+juju backups create --filename backup-19.tgz
+```
+
+Note that creating a backup may take a long time.
+
+
+### Managing Backups
+
+Each time a backup is created, it is also stored on the state-server for
+safekeeping, along with associated metadata giving it a unique id value and a 
+timestamp. This means that you can manage backups from whatever client you can
+connect from, and fetch previous backups if the originally downloaded file has
+gone astray. For managing backups, the `juju backups` command has an additional 
+set of subcommands:
+
+### juju backups download 
+
+usage: `juju backups download [options] <ID>`
+
+The `download` command will fetch the backup specified from the environment 
```

Comment:
Previously it was stated that the creation of a backup will already download the file. I feel the upload/download/storing of these files needs to be explained more.